doc/book/src/installation.md

```diff
@@ -1,30 +1,34 @@
 # Installation
 
+## Requirements
+
+Maelstrom currently only supports Linux.
+
+...presumably there are also kernel version requirements?
+
+## Installed Binaries
+
 Maelstrom consists of the following binaries:
   - [`cargo-maelstrom`](cargo-maelstrom.md): A Rust test runner. This can be run in
     standalone mode &mdash; where tests will be executed on the local machine
     &mdash; or in clustered mode. In standalone mode, no other Maelstrom
     binaries need be installed, but in clustered mode, there must be a broker
     and some workers available on the network.
-  - [`maelstrom-broker`](broker.md): The Maelstrom cluster scheduler. There must be one of
```

Comment:
I thought it was weird to call the `broker` the "cluster scheduler" here. If it's a scheduler, why isn't it called `maelstrom-scheduler`?

Reply:
Well, it contains the scheduler. :-)

Any suggestions for how to better explain it?

It now reads:

> The `maelstrom-broker` is the coordinator and scheduler for a Maelstrom
cluster. In order to have a cluster, there must be a broker. The broker must be
started before clients and workers, as the clients and workers connect to the
broker, and will exit if they can't establish a connection.